flink-yarn-tests/src/test/java/org/apache/flink/yarn/YARNITCase.java

```diff
@@ -113,6 +120,30 @@ public void testPerJobMode() throws Exception {
 
 					assertThat(jobResult, is(notNullValue()));
 					assertThat(jobResult.getSerializedThrowable().isPresent(), is(false));
+
+					final FileSystem fs = FileSystem.get(getYarnConfiguration());
+					String suffix = ".flink/" + applicationId.toString() + "/" + flinkUberjar.getName();
+
+					Path uberJarHDFSPath = new Path(fs.getHomeDirectory(), suffix);
+					FileStatus fsStatus = fs.getFileStatus(uberJarHDFSPath);
+					Assert.assertEquals(5, fsStatus.getReplication());
+
+					Path appPath = uberJarHDFSPath.getParent();
+					FileStatus[] fileStatuses = fs.listStatus(appPath, new PathFilter() {
+						@Override
+						public boolean accept(Path path) {
+							return path.getName().endsWith("taskmanager-conf.yaml");
+						}
+					});
+
+					final int replication = getYarnConfiguration().getInt(DFSConfigKeys.DFS_REPLICATION_KEY,
+						DFSConfigKeys.DFS_REPLICATION_DEFAULT);
+
+					// All of the task manager confs are used by single TM, the replica number should be DFS_REPLICATION_DEFAULT.
+					for (FileStatus fileStatus : fileStatuses) {
+						Assert.assertEquals(replication, fsStatus.getReplication());
```

Comment:
Is it a typo? It should be `fileStatus.getReplication()`.

Reply:
Yes, you are right.